lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/table/ColumnModel.json

```diff
@@ -41,6 +41,11 @@
 			"items": {
 				"type": "string"
 			}
+		},
+		"facetType": {
+			"description": "The type of faceting applied to this column",

```

Comment:
Set to one of the enumerated values to indicate this column should be treated as a facet. (also describe each enum).